src/components/ProfileValidationSteps/ProfileFieldsEditor.tsx

```diff
@@ -1,5 +1,7 @@
+import { Box } from '@mui/material'
```

Comment:
needs cleanup. Need to move from bootstrap to material controls and add validation